automation/DataAggregator/BaseAggregator.py

```diff
@@ -93,10 +93,9 @@ def shutdown(self):
 
     def drain_queue(self):
         """ Ensures queue is empty before closing """
-        self.sock.close()
```

Comment:
Is that change intentional?

Reply:
Yes, we end up calling `sock.close()` twice. Once here and once in `BaseAggregator::close`.